test/scripts/e2e_subs/tealprogs/abi-demo.json

```diff
@@ -0,0 +1,216 @@
+{
+    "name": "demo-abi",
+    "appId": null,
```

Comment:
nit: seems ok to remove `"appId": null,"` this line?

Reply:
Will need to figure out what to do with this when the Python SDK gets updated to reflect the final ARC-04